deploy/packaging/linux.xml

```diff
@@ -836,6 +836,12 @@ rm -f /etc/ld.so.conf.d/mdsplus.conf 2&gt;/dev/null
  </packages>
 
  <!-- dist first -->
+  <external_packages dist="Ubuntu24">
```

Comment:
Looks good, after we drop support for Ubuntu18/DebianBuster we can change the `debian` default below and remove all these overrides